src/com/perl5/lang/perl/util/PerlRunUtil.java

```diff
@@ -83,6 +100,82 @@ public static GeneralCommandLine getPerlCommandLine(@NotNull Project project, @N
 		return commandLine;
 	}
 
+	private static List<String> getPerlLibraries(@NotNull Project project, PerlConfiguration runProfile, @Nullable VirtualFile scriptFile)
+	{
+		if (PlatformUtils.isIntelliJ())
+		{
+			// try to get used SDK
+			Sdk sdk;
+			if (runProfile != null && runProfile.isUseAlternativeSdk())
+			{
+				sdk = ProjectJdkTable.getInstance().findJdk(runProfile.getAlternativeSdkPath());
+			}
+			else
+			{
+				Module moduleForFile = scriptFile == null ? null : ModuleUtilCore.findModuleForFile(scriptFile, project);
+
+				sdk = getModuleSdk(moduleForFile);
+				if (sdk == null)
+				{
+					// found in project
+					sdk = ProjectRootManager.getInstance(project).getProjectSdk();
+					if (sdk == null || sdk.getSdkType() != PerlSdkType.getInstance())
+					{
+						// looking for any perl module in project
+						for (Module module : ModuleManager.getInstance(project).getModules())
+						{
+							sdk = getModuleSdk(module);
+							if (sdk != null && sdk.getSdkType() == PerlSdkType.getInstance())
+							{
+								break;
+							}
+						}
+					}
+				}
+			}
+
+			if (sdk != null && sdk.getSdkType() == PerlSdkType.getInstance())
+			{
+				List<String> incPaths = PerlSdkType.getInstance().getINCPaths(sdk.getHomePath());
+				List<String> list = new ArrayList<>();
+				for (VirtualFile file : sdk.getRootProvider().getFiles(OrderRootType.CLASSES))
+				{
+					// add only paths that not already exist
+					String path = file.getPath().replace("\\", "/");
+					if (!incPaths.contains(path)) {
+						list.add(path);
+					}
+				}
+
+				return list;
+			}
+			else
+			{
+				return new ArrayList<>();
+			}
+		}
+		else
+		{
+			return new ArrayList<>();
+		}
+	}
+
+	@Nullable
+	private static Sdk getModuleSdk(Module module)
+	{
+		if (module == null)
```

Comment:
This duplicates com.perl5.lang.perl.util.PerlRunUtil#getModuleSdkPath
You should keep code DRY and adjust getModuleSdkPath to use your code